### ISSUE TITLE
Running on Linux?

### ISSUE THREAD

**Issue by dvdmuckle** [electroduck/display16#1]
I'm guessing this program wouldn't be able to run on Linux considering the inclusion of the `Windows.h` header file. That being said, to what extent is the header file utilized? I don't have much experience programming on Windows, let alone using this particular header file.

**Comment by electroduck** [comment]
Yeah, this program would require significant modifications, really a whole recoding, to run on Linux. It uses the Command Prompt as an array of character attributes (character code & color) rather than a stream output device, which I don't think Linux supports. The terminal in Linux is a teletypewriter style system with special escape codes to set colors. That's a totally different system.
There are some Linux programs already existing that have similar functionality.  mplayer is on one. In my experience, though, the performance is quite poor because of the need to output escape codes to a stream instead of just setting a bunch of values in an array and pushing it directly to the console.

**Comment by electroduck** [comment]
The converter program could be made to run on Linux, though. Just not the display programs. This would simply require splitting the conversion code into a seperate C and H files from the display code.

**Comment by dvdmuckle** [comment]
That's kinda what I thought. Wonder if looking into how ncurses does terminal displays might help.
